gradle/deploy.sh

```diff
@@ -15,12 +15,10 @@ elif [ $CIRCLE_TAG ]; then
   openssl aes-256-cbc -d -in gradle.properties.enc -out gradle.properties -k "$KEY" -md sha256
   case "$CIRCLE_TAG" in
   *-rc\.*)
-    ./gradlew -Prelease.disableGitChecks=true -Prelease.useLastTag=true candidate bintrayUpload $SWITCHES -x release -x bintrayPublish
-    ./gradlew -Prelease.disableGitChecks=true -Prelease.useLastTag=true :micrometer-core:bintrayPublish $SWITCHES -x bintrayUpload
+    ./gradlew -Prelease.disableGitChecks=true -Prelease.useLastTag=true candidate publishAllPublicationsToMilestoneRepository $SWITCHES -x release -x artifactoryPublish -x bintrayUpload -x bintrayPublish
```

Comment:
Nit: We could make `candidate` depend on the publish task in the build so the command here is simpler. I know it sounds a bit superfluous with all the skipped tasks here... We could follow-up getting rid of those with an update to the spring-release-plugin we're using.

Reply:
I was thinking on it too but eventually I did not do it since this way it is easier to test things locally and gives more information about what is happening. What do you think about cleaning this-up with the release plugin cleanup (that should also eliminate the excluded tasks)?

Reply:
Yes, I think it's fine to defer to when we change the release plugin. I think all of this is vetting the process before we bake it into a released version of the plugin.